Jenkinsfile

```diff
@@ -66,6 +66,29 @@ for(i = 0; i < buildTypes.size(); i++) {
     }
 }
 
+builds.ath = {
+    node("linux") {
+        // Just to be safe
+        deleteDir()
+        def fileUri
+        def metadataPath
+        dir("sources") {
+            checkout scm
+            withMavenEnv(["JAVA_OPTS=-Xmx1536m -Xms512m",
+                          "MAVEN_OPTS=-Xmx1536m -Xms512m"]) {
+                sh "mvn -DskipTests -am -pl war package -Dmaven.repo.local=${pwd tmp: true}/m2repo -s settings-azure.xml"
+            }
+            dir("war/target") {
+                fileUri = "file://" + pwd() + "/jenkins.war"
```

Comment:
As noted in #3347, this is unsafe: you cannot in general construct a `file`-protocol URL merely by appending strings; you need to do URL encoding. Currently there is not a Pipeline step to translate a file path to a URL, so I would rather recommend fixing the `runATH` variable to accept a local file path instead of a URL, perhaps as a differently-named option.